railties/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## (Unreleased)
 
+*   Make generated application_system_test_case.rb files avoid
+    `ChildProcess::LaunchError` when parallel tests are enabled.
+
+    *Richard Macklin*
```

Comment:
Since this is a bug fix we don't need a changelog entry.

Reply:
Sorry about that; I'll remove this with the other changes.